saq/job.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
+
 import dataclasses
 import enum
 import typing
```

Comment:
let's refactor everywhere that uses typing to the convention

import typing as t